rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/Row.java

```diff
@@ -94,4 +97,22 @@ public void add(Field field) {
         }
         return nonPkFields;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Row row = (Row) o;
+        List<Field> oFields = row.fields;
+
+        if (CollectionUtils.isNotEmpty(this.fields) && CollectionUtils.isNotEmpty(oFields)) {
+            return StringUtils.equals(this.fields.toString(), oFields.toString());
```

Comment:
why equals toString? Arrays.equals?

Reply:
i will change to use Arrays.equals in the new commit

Reply:
the fields is a List, not an array